app/posts/register-trainee-teachers/2021-08-03-highest-academic-qualification.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Removing the highest academic qualification question
+description: Why we don’t think we need this question and an alternate design if we do.
+date: 2021-08-03
+---
+{% from "figure/macro.njk" import appFigure with context %}
+
+For trainees on undergraduate courses the provider has been asked for the trainee’s highest academic qualification.
+
+The question was causing users confusion. This is seems to be because it’s inconsistent with how users think about academic qualifications. A teacher training provider thinks about a trainee’s qualifications as the measure of whether a trainee is qualified and often what their ‘tariff points’ value is. In research sessions users tried to give the trainee’s actual qualifications. When users found they could only provide a single qualification most users re-read the question and were able to give the answer the service wanted.
+
+The options we gave as answers confused the users too. Its a long list that includes qualifications that were too low level to be relevant.
+
+## DTTP data
+
+This spreadsheet of [EY Undergraduate by 'Highest qualification on entry'](https://docs.google.com/spreadsheets/u/1/d/1fb2ialqhFKxVh8WZ8yeoeKkJrtcJNnUm/edit#gid=1730684280) has 67 records for the last 2 years.
+
+- 9 don’t have any highest qualification data
+- 58 do have a highest qualification
+
+Of the 58 records:
+
+- 26 Any combinations of GCE 'A'/SQA 'Higher'/SQA 'Advanced Higher' & GNVQ/GSVQ or NVQ/SVQ at level 3
+- 11 'A' level equivalent qualification not elsewhere specified
+- 11 Other higher qualification
+- 5 ONC or OND (including BTEC & SQA equivalents)
+- 3 Foundation course at FE level
+- 1 NVQ/SVQ level 3
+- 1 Baccalaureate
+
+As (roughly rounded) percentages:
+
+- 45% Any combinations of GCE 'A'/SQA 'Higher'/SQA 'Advanced Higher' & GNVQ/GSVQ or NVQ/SVQ at level 3
+- 19% 'A' level equivalent qualification not elsewhere specified
+- 19% Other higher qualification
+- 9% ONC or OND (including BTEC & SQA equivalents)
+- 5% Foundation course at FE level
+- 2% NVQ/SVQ level 3
+- 2% Baccalaureate
+
+Using this page on [What qualification levels mean](https://www.gov.uk/what-different-qualification-levels-mean) you can group them into levels:
+
+- 75% are level 3
+- 19% are ‘other’
+- 5% Foundation course at FE level
+- 2% Baccalaureate
+
+Given that the data is too high level to be of any value to the provider, or trainee (for example, as part of their record) and the number of trainees on undergraduate courses through SCITTs is tiny compared to the number going through HEIs and the data that is captured is 19% ‘other’ there's not much value this can be contributing as a statistical measure it seems most sensible to remove the question.
```

Comment:
"Given that the data is too high level to be of any value to the provider" - I think this is too strong. We believe it's unlikely to be of value as it seems too high level.

Reply:
First sentence is rather complex.